src/include/cell_inline.h

```diff
@@ -951,13 +958,12 @@ __wt_cell_unpack_safe(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CE
                  * prepared id in WT_CELL_TS_DURABLE_STOP.
                  */
                 WT_ASSERT(session,
-                  LF_ISSET(WT_CELL_TS_START) && LF_ISSET(WT_CELL_TS_DURABLE_START) &&
-                    LF_ISSET(WT_CELL_TS_STOP) && LF_ISSET(WT_CELL_TS_DURABLE_STOP));
+                  LF_ISSET(WT_CELL_TS_START) && LF_ISSET(WT_CELL_TS_STOP) &&
```

Comment:
I think we should also remove LF_ISSET(WT_CELL_TS_START) from the assert. It can be none.